src/node/transaction.cpp

```diff
@@ -6,7 +6,6 @@
 #include <consensus/validation.h>
 #include <net.h>
 #include <net_processing.h>
-#include <txmempool.h>
```

Comment:
This is actually used

Reply:
Oh, I'll investigate shortly! Any more details?